README.md

```diff
@@ -1,8 +1,14 @@
-This is the Github-hosted branch of edx-platform that is behind
-http://docs.edx.org
+## Overview
+This is the Github-hosted and tied to the master branch that is behind
+http://docs.edx.org. 
 
+## Search
 
-The search.html page is powered by cse.google.com, configured here:
-https://cse.google.com/cse/setup/basic?cx=003935656615667250026:jtodz7s8c9a
+The `search.html` page is powered by Google Programmable Search Engine and can be configured [here](https://cse.google.com/cse/setup/basic?cx=003935656615667250026:jtodz7s8c9a):
 
-The current list of sites that should be configured there is in search-sites.txt.
+The current list of sites that should be configured there is in `search-sites.txt`.
+
+## Contribution
+If you want to contibute, please open a pull request against master and tag [community-engineering](https://github.com/orgs/edx/teams/community-engineering) team for a review. There is no extra release process and as soon as you merge to master, the changes will be reflected staright away on production. 
```

Comment:
Can people who are not edX employees see that team information? Is there an email address they can reach out to?

Reply:
I don't think teams are public. At least hitting it when logged out of GitHub doesn't show up.

Reply:
Good call, let me check. 
if they are not public, how we should reword it to indicate that they are the owners and should be tagged for the changes?

Reply:
Yes, 404 for non-edx team member.

Reply:
I have removed the _team_ reference/link from the document now.